src/validation.cpp

```diff
@@ -2834,12 +2833,11 @@ bool Chainstate::ConnectTip(BlockValidationState& state, CBlockIndex* pindexNew,
     const CBlock& blockConnecting = *pthisBlock;
     // Apply the block atomically to the chain state.
     const auto time_2{SteadyClock::now()};
-    time_read_from_disk_total += time_2 - time_1;
     SteadyClock::time_point time_3;
-    LogPrint(BCLog::BENCH, "  - Load block from disk: %.2fms [%.2fs (%.2fms/blk)]\n",
-             Ticks<MillisecondsDouble>(time_2 - time_1),
-             Ticks<SecondsDouble>(time_read_from_disk_total),
-             Ticks<MillisecondsDouble>(time_read_from_disk_total) / num_blocks_total);
+    // When adding aggregate statistics in the future, keep in mind that
+    // num_blocks_total may be zero until the ConnectBlock() call below.
```

Comment:
Fwiw I think this warning is useful because many of the bench logs use `/ num_blocks_total` and `ConnectTip` has an `assert(num_blocks_total > 0)` just a few lines down.

Reply:
We also call `ConnectTip()` from `TestBlockValidity()` and `VerifyDB()`. Do you know how the counter is supposed to behave in those cases?

Reply:
Happy to review such a change, but I barely remember making the original PR and am not very familiar with the bench logging.